src/main/java/com/rultor/agents/daemons/EndsDaemon.java

```diff
@@ -70,15 +76,23 @@
     public static final String HIGHLIGHTS_PREFIX = "RULTOR: ";
 
     /**
+     * The profile to check.
```

Comment:
@essobedo lets adjust this to just say `The profile of the run.` Even though we're "checking" it for the deprecation notice this should rather say what this field holds, not what it's used for imo.